examples-gallery/plot_parallel_park.py

```diff
@@ -252,6 +258,8 @@ def frame(i):
 
 fig, title_text, lines, Pr_path, Pf_path = frame(0)
 
+time = list(time)
+time[-3:] = [time[-1]]*3
```

Comment:
What are you doing with these lines?

Reply:
> What are you doing with these lines?

Without them I did not get the time showing on the animation to end at the correct time. It ended ar 29.88 sec (if I recall correctly) instead of at 30 sec.